code/datums/components/storage/concrete/extract_inventory.dm

```diff
@@ -0,0 +1,35 @@
+/datum/component/storage/concrete/extract_inventory
+	max_combined_w_class = WEIGHT_CLASS_TINY * 3
+	max_items = 3
+	insert_preposition = "in"
+//These need to be false in order for the extract's food to be unextractable
+//from the inventory
+	attack_hand_interact = FALSE
+	quickdraw = FALSE
+	can_transfer = FALSE
+	drop_all_on_deconstruct = FALSE
+	locked = TRUE //True in order to prevent messing with the inventory in any way other than the specified ways on reproductive.dm
+	rustle_sound = FALSE
+	silent = TRUE
+	var/obj/item/slimecross/reproductive/parentSlimeExtract
+
+/datum/component/storage/concrete/extract_inventory/Initialize()
```

Comment:
Shouldnt this be ComponentInitialize()? Haven't worked with components at a fundamental level.

Reply:
also am not to well versed with components honestly

Reply:
also unsure cause i also have not done much with components outside this so i cannot really speak here

Reply:
hmmm no it shouldn't componentinitialize() gets called by /atom/proc/Initialize() so it will not get called by a datum Initialize

Reply:
No, component initialize is it's own thing